exetera/core/fields.py

```diff
@@ -23,6 +23,15 @@
 
 class HDF5Field(Field):
     def __init__(self, session, group, dataframe, write_enabled=False):
+        """
+        Construct a HDF5 file based Field. This construction is not used directly, rather, should be called from
```

Comment:
Constructor for Fields based on HDF5 data store. This initializer should only be called by Field types that are subclasses of HDF5Field, and never directly by users of ExeTera.